pkg/mock/mockcluster/mockcluster.go

```diff
@@ -299,6 +299,23 @@ func (mc *Cluster) AddRegionStore(storeID uint64, regionCount int) {
 	mc.PutStore(store)
 }
 
+// AddWitnessStore adds store with specified count of witness.
+func (mc *Cluster) AddWitnessStore(storeID uint64, witnessCount int) {
+	stats := &pdpb.StoreStats{}
+	stats.Capacity = defaultStoreCapacity
+	stats.UsedSize = uint64(witnessCount) * defaultRegionSize
+	stats.Available = stats.Capacity - uint64(witnessCount)*defaultRegionSize
```

Comment:
I have a confusion, witness does not occupy storage space, so why is it calculated here?

Reply:
oh, by mistake